cad/scripts/_common.py

```
@@ -1153,6 +1153,68 @@ def _git_sha() -> str:
+    the count is deterministic per commit, monotonic between releases, and the
+    release commit is simply the last candidate.  Stamped only when a drawing
+    task actually runs -- git state is in no cache key or file_dep, so a commit
+    never rebuilds anything; a restored sheet keeps the id of the build that
+    made it.
+    """
+    import subprocess
+
+    import _config
+
+    def _git(*args: str) -> str:
+        return subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)